backend/db/seeds.rb

```diff
@@ -1,16 +1,19 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the rails db:seed command (or created alongside the database with db:setup).
-#
-# Examples:
-#
-#   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
-#   Character.create(name: 'Luke', movie: movies.first)
 question_list = [
-    ['What sweet food made by bees using nectar from flowers?', 'Honey', 'easy'],
-    ['Name the school that Harry Potter attended?', 'Hogwarts', 'medium'],
-    ['What\'s the capital of Zimbabwe?', 'Harare', 'hard' ]
+  ['What sweet food made by bees using nectar from flowers?', 'Honey', 'easy'],
+  ['Name the school that Harry Potter attended?', 'Hogwarts', 'medium'],
+  ['What\'s the capital of Zimbabwe?', 'Harare', 'hard' ]
+]
+
+token_list = [
+  ['dfYGLxuSEbGW6gh65KUX2Qna'],
+  ['vE9M8DMiJYqEHbduqX2huPYJ'],
+  ['s7XsPnTuPMJRwytf3vnNw7GA']
```

Comment:
It seems like you are planning to store these tokens as a populated data for client records. How are you going to match it with the whitelisted tokens [here](https://github.com/ninariccimarie/quiz-master/pull/6/commits/0f20640c9f3718acdc2d0178346ed54113e1dea9#diff-1432a41783ad66075e0c3b15bd94c02aR26)?